config/secrets.yml

```diff
@@ -11,12 +11,15 @@
 # if you're sharing your code publicly.
 
 development:
-  secret_key_base: 36e869f0f24abe0e14702e47398cf31e433c0c39622b5c72bff217bd5a150505caf5e95336f5200b767645432909e66fc47caca14ad07e4fa8cfb68883a05dde
+  secret_key_base: ca9f11ea3b721ecaa2916e7a5886f55a9e25e31f0896172f250a026d698441a08f9026c0cbe79a85c0872b2802e2a2fdf25d4bea573f45878c28d3eef22cbc9e
 
 test:
-  secret_key_base: c1212ec1ca179a3b14104cbd3fd2df298bfcd4596f9303fce590c9a76e66d9b15c97514264dec8689950e93b8fe937937b16da78221291e74427014b78b69bdf
+  secret_key_base: fd62c9d3e4bf141e24a91e3a4430d91576e7747ceee28240d232f86fd1e8b654064a6a8430bd3d026bc4e65a618960dd485d21303a48cbcb88a5c637082cba75
```

Comment:
Can we also use `figaro` for these? I know it is just for `development` and `test`, but we don't want to give them a clue.